Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1899,6 +1983,117 @@ def get_notable_sequence_event_types(client: Client, args: Dict[str, str]) -> Tu
     return human_readable, entry_context, sequence_event_types_raw_data
 
 
+def list_incidents(client: Client, args: Dict[str, str]):
+    incident_ids = argToList(args.get('incident_id'))
+    query = args.get('query')
+    incident_type = args.get('incident_type')
+    priority = args.get('priority')
+    status = args.get('status')
+    limit = args.get('limit', 50)
+    page_size = args.get('page_size', 25)
+    page_number = args.get('page_number', 1)
+    username = args.get('username')
+
+    if incident_ids and client.username == TOKEN_INPUT_IDENTIFIER and not username:
+        raise ValueError('The username argument is necessary be for this command if the instance configured by api key')
+
+    incidents = []
+    raw_response = dict()
+
+    if incident_ids:
+        for incident_id in incident_ids:
+            raw_response = client.get_single_incident(incident_id, username)
+            incidents.append(format_single_incident(raw_response))
+
+    else:
+        if any([query, incident_type, priority, status]):
+            raw_response = client.get_list_incidents(query, incident_type, priority, status, limit, page_size,
+                                                     page_number)
+            for incident in raw_response['incidents']:
+                incidents.append(format_single_incident(incident))
+        else:
+            return_error('One of the following params is a must: query, incident_type, priority, status')
+
+    entry_context = {'Exabeam.Incident(val.incidentId && val.incidentId === obj.incidentId)': incidents}
+
+    human_readable = tableToMarkdown('Incidents list:', incidents)
+
+    return human_readable, entry_context, raw_response
+
+
+def fetch_incidents(client: Client, args: Dict[str, str]) -> Tuple[list, dict]:
+
+    last_run = demisto.getLastRun()
+    demisto.debug(f"Last run before the fetch run: {last_run}")
+    start_time, end_time = get_fetch_run_time_range(
+        last_run=last_run,
+        first_fetch=args.get('first_fetch', '3 days'),
+        look_back=1,
+        date_format='%Y-%m-%dT%H:%M:%S.%f',
+    )
+
+    demisto.debug(start_time, end_time)
```

Comment:
what's the 2nd argument? Perhaps it's better to debug a string that says what's happening, e.g. `f"fetching incidents between {start_time=}, {end_time=}`